hieradata/clients/cert-ci.yaml

```
@@ -98,6 +99,7 @@ profile::jenkinscontroller::jcasc:
           architecture: amd64
           labels:
             - windows
+            - docker
```

```suggestion
            - docker-windows
```

Otherwise it will a good recipe for failure while trying to spin up docker linux container on a docker windows engine